server/base/src/main/java/org/apache/accumulo/server/conf/NamespaceConfiguration.java

```diff
@@ -167,4 +167,9 @@ public synchronized void invalidateCache() {
     // to see if it happened to be created so we could invalidate its cache
     // but I don't see much benefit coming from that extra check.
   }
+
+  @Override
+  public long getUpdateCount() {
+    return Math.max(parent.getUpdateCount(), getPropCacheAccessor().getZooCache().getUpdateCount());
```

Comment:
This should be sum of parent and self, not max.

Reply:
Nice catch.